spyder/plugins/variableexplorer/widgets/main_widget.py

```diff
@@ -413,76 +405,44 @@ def on_section_conf_change(self, section):
 
     # ---- Stack accesors
     # ------------------------------------------------------------------------
-    def update_finder(self, nsb, old_nsb):
-        """Initialize or update finder widget."""
-        if self.finder is None:
-            # Initialize finder/search related widgets
-            self.finder = QWidget(self)
-            self.text_finder = NamespacesBrowserFinder(
-                nsb.editor,
-                callback=nsb.editor.set_regex,
-                main=nsb,
-                regex_base=VALID_VARIABLE_CHARS)
-            self.finder.text_finder = self.text_finder
-            self.finder_close_button = self.create_toolbutton(
-                'close_finder',
-                triggered=self.hide_finder,
-                icon=self.create_icon('DialogCloseButton'),
-            )
-
-            finder_layout = QHBoxLayout()
-            finder_layout.addWidget(self.finder_close_button)
-            finder_layout.addWidget(self.text_finder)
-            finder_layout.setContentsMargins(0, 0, 0, 0)
-            self.finder.setLayout(finder_layout)
-
-            layout = self.layout()
-            layout.addSpacing(1)
-            layout.addWidget(self.finder)
-        else:
-            # Just update references to the same text_finder (Custom QLineEdit)
-            # widget to the new current NamespaceBrowser and save current
-            # finder state in the previous NamespaceBrowser
-            if old_nsb is not None:
-                self.save_finder_state(old_nsb)
-            self.text_finder.update_parent(
-                nsb.editor,
-                callback=nsb.editor.set_regex,
-                main=nsb,
-            )
-
     def switch_widget(self, nsb, old_nsb):
         """
         Set the current NamespaceBrowser.
 
         This also setup the finder widget to work with the current
         NamespaceBrowser.

```

Comment:
```suggestion
        """Set the current NamespaceBrowser."""
```

The second part of this docstring is no longer correct.